controllers/keystoneapi_controller.go

```diff
@@ -1201,7 +1215,19 @@ func (r *KeystoneAPIReconciler) generateServiceConfigMaps(
 		"fernetMaxActiveKeys": instance.Spec.FernetMaxActiveKeys,
 	}
 
+	templateParameters["KeystoneEndpointPublic"], _ = instance.GetEndpoint(endpoint.EndpointPublic)
```

Comment:
since this is essentially an api do we need to version it?

Reply:
Or under a top-level version, e.g I _think_ this would work:
```
v1 := make(map[string]interface{})
v1["KeystoneEndpointPublic"], _ = instance.GetEndpoint(endpoint.EndpointPublic)
...
templateParameters["v1"] = v1
```

Reply:
done